tests/core.py

```diff
@@ -1625,15 +1629,26 @@ def test_chart_data_template(self):
         session.add(chart2)
         session.add(chart3)
         session.commit()
-        chart1_id = session.query(Chart).filter(Chart.label=='insecure_chart').first().id
+        chart1_id = session.query(Chart).filter(Chart.label == 'insecure_chart').\
+            first().id
```

Comment:
This change and several others are not related to changing the config, but are necessary to pass the new flake8 test.

Reply:
Please use () instead